### ISSUE TITLE
Verification Process Slow?

### ISSUE THREAD

**Issue by NickB23** [Creepsky/creepMiner#210]
Hi. I'm mining with 3,59TB on my raspberry pi 3 model b with creepMiner 1.6.3. Read times are incredibly good for the raspberry pi, however the verification process never gets any further then 0,08% before the next block.
![image 2017-10-01 at 9 48 25 am public](https://user-images.githubusercontent.com/16944740/31052758-fda0cdbc-a68d-11e7-9782-e8ac82c0ba73.png)

My mining.conf looks like this:
```
{
    "logging" : {
        "config" : "information",
        "general" : "information",
        "logfile" : false,
        "miner" : "information",
        "nonceSubmitter" : "information",
        "output" : {
            "dirDone" : true,
            "lastWinner" : true,
            "nonceConfirmed" : true,
            "nonceFound" : true,
            "nonceFoundTooHigh" : false,
            "nonceOnTheWay" : true,
            "nonceSent" : true,
            "plotDone" : true
        },
        "outputType" : "terminal",
        "path" : "",
        "plotReader" : "information",
        "plotVerifier" : "all",
        "progressBar" : {
            "fancy" : false,
            "steady" : true
        },
        "server" : "fatal",
        "session" : "error",
        "socket" : "off",
        "useColors" : true,
        "wallet" : "information"
    },
    "mining" : {
        "bufferChunkCount" : 8,
        "getMiningInfoInterval" : 3,
        "intensity" : 2,
        "maxBufferSizeMB" : 512,
        "maxPlotReaders" : 0,
        "passphrase" : {
            "algorithm" : "",
            "decrypted" : "",
            "deleteKey" : false,
            "encrypted" : "",
            "iterations" : 0,
            "key" : "",
            "salt" : ""
        },
        "plots" : [
            {
                "path" : "/media/pi/SEAGATE/Burst/plots",
                "type" : "sequential"
            }
        ],
        "rescanEveryBlock" : false,
        "submissionMaxRetry" : 5,
        "targetDeadline" : 2592000,
        "timeout" : 60,
        "urls" : {
            "miningInfo" : " MINING URL POOL HERE",
            "submission" : "MINING URL POOL HERE",
            "wallet" : "https://wallet1.burstnation.com:8125"
        },
        "useInsecurePlotfiles" : false,
        "wakeUpTime" : 0,
        "walletRequestRetryWaitTime" : 3,
        "walletRequestTries" : 5
    },
    "webserver" : {
        "credentials" : {
            "hashed-pass" : "",
            "hashed-user" : "",
            "pass" : "",
            "user" : ""
        },
        "start" : true,
        "url" : "http://localhost:8080"
    }
}
```
Am I doing something wrong or is this expected behaviour?

**Comment by Creepsky** [comment]
This is a display bug due to 32 bit, fixed in 1.7.0+.

**Comment by cuthulino** [comment]
I can submit it is displaying the values now.
